doc/misc/npm-faq.md

```diff
@@ -9,17 +9,13 @@ npm-faq(7) -- Frequently Asked Questions
 
 This command will set the npm docs to open in your default web browser rather than `man`.
 
-## It didn't work.
```

Comment:
These two points can be combined to one and don't need to take such an annoyed tone. It's one of the first things the user will read - no need to make them feel bad for using the FAQ page provided for the very purpose they're using it.